docs/v1/source/includes/_simplifiedAPI.md

```diff
@@ -8,7 +8,7 @@ Accounts Functions
 // Accounts Simplified API Examples:
 
 augur.accounts.approveAugur({
-  address: "0x0000000000000000000000000000000000000b0b",
+  address: "0x0000000000000000000000000000000000000b0b", 
```

Comment:
whitespace!

Reply:
It's just in the docs md files. Shouldn't be noticeable in the actual docs pages.